scraper/scraper.py

```diff
@@ -199,4 +201,7 @@ def send_to_database(course):
     print(course.code + " - " + course.name)
 
 if __name__ == '__main__':
+    # TODO remove this once actual database functions are implemented & available
```

Comment:
All the changes in this file besides this and the `import db` above are just reformatting